draft-ietf-scitt-architecture.md

```diff
@@ -373,280 +369,155 @@ Most of the details of the Receipt's contents are specified in the COSE Signed M
+The mechanisms for how Transparency Services obtain identity documents is out-of-scope of this document.
 
-## Signed Statement Envelope
+The `kid` header parameter MUST be present when the `x5t` header parameter is not present.
+Key discovery protocols are out-of-scope of this document.
 
-Signed Statements are CBOR encoded {{-CBOR}} and protected by CBOR Object Signing and Encryption (COSE {{-COSE}}).
-Signed Statements contain a protected, an unprotected header and a payload.
+The protected header of a Signed Statement and a Receipt MUST include the `CWT Claims` header parameter as specified in {{Section 2 of CWT_CLAIMS_COSE}}.
+The `CWT Claims` value MUST include the `Issuer Claim` (Claim label 1) and the `Subject Claim` (Claim label 2) {{IANA.cwt}}.
 
-All Signed Statements MUST include the following protected headers:
+A Receipt is a Signed Statement, with addition claims in its protected header related to verifying the inclusion proof in its unprotected header. See {{-COMETRE}}.
 
-- **algorithm** (label: `1`): Asymmetric signature algorithm used by the Issuer of a Signed Statement, as an integer.<br>
-  Example: `-35` is the registered algorithm identifier for ECDSA with SHA-384, see [COSE Algorithms Registry](#IANA.cose).
-- **Key ID** (label: `4`): Key ID, as a bytestring
-- **CWT_Claims** (label: `15` pending {{CWT_CLAIMS_COSE}}): A CWT representing the Issuer (`iss`) making the statement, and the Subject (`sub`) to correlate a collection of statements about an Artifact.
-  Additional {{CWT_CLAIMS}} MAY be used, while `iss` and `sub` MUST be provided
-  - **iss** (CWT_Claim Key `1`): The Identifier of the signer, as a string<br>
-    Example: `https://software.vendor.example`
-  - **sub** (CWT_Claim Key `2`): The Subject to which the Statement refers, chosen by the Issuer<br>
-    Example: `github.com/opensbom-generator/spdx-sbom-generator/releases/tag/v0.0.13`
-- **Content type** (label: `3`): The media type of the payload, as a string.<br>
-  Example: `application/spdx+json` as the media type of SDPX in JSON encoding
+Figure {{fig-signed-statement-cddl}} illustrated a normative CDDL definition for of the protected header for Signed Statements and Receipts.
 
-In CDDL {{-CDDL}} notation, a Signed_Statement is defined as follows:
+Everything that is optional in the following CDDL can potentially be discovered out of band and Registration Policies are not assured on the presence of these optional fields.
+A Registration Policy that requires an optional field to be present MUST reject any Signed Statements or Receipts that an invalid according to the policy.
 
 ~~~ cddl
-Signed_Statement = COSE_Sign1_Tagged
 
-COSE_Sign1_Tagged = #6.18(COSE_Sign1)
+Signed_Statement = #6.18(COSE_Sign1)
+Receipt = #6.18(COSE_Sign1)
 
 COSE_Sign1 = [
   protected   : bstr .cbor Protected_Header,
   unprotected : Unprotected_Header,
-  payload     : bstr,
+  payload     : bstr / nil,
```

Comment:
I thought when a `Payload` is a `hash` it cannot be `nil` ?

Reply:
For a receipt, the verifiable data structure defines what this is... for RFC9162, the payload is "computed from the inclusion proof".